Assets/Scripts/Player/Movement.cs

```diff
@@ -14,25 +17,44 @@ public class Movement : MonoBehaviour
     void Start()
     {
         _rb = GetComponent<Rigidbody2D>();
+        _currentMoveSpeed = _movementSpeed;
     }
 
     void Update()
     {
-        _input = new Vector2(Input.GetAxis("Horizontal"), Input.GetAxis("Vertical"));
+        _input = new Vector2(Input.GetAxisRaw("Horizontal"), Input.GetAxisRaw("Vertical"));
+         _input.Normalize();
```

Comment:
Try to not add any unnecessary lines, you can call the Normalize method after creating the new Vector2 immediately on line 25